internal/controller/constants.go

```diff
@@ -12,6 +12,10 @@ const (
 	/*** application server configuration file ***/
 	// OLSConfigName is the name of the OLSConfig configmap
 	OLSConfigCmName = "olsconfig"
+	// OLSRedisCACmName is the name of the OLS redis server TLS ca certificate configmap
+	OLSRedisCACmName = "openshift-service-ca.crt"
```

Comment:
OLSRedisCACmName - Do we want OLS prefix for every variable name? Could this be RedisCAConfigMap ? @vishnuchalla @raptorsun WDYT ?

Reply:
We can remove the `OLS` prefix, if everyone has it, it is no longer a distinguishing prefix. 
If we will not have other application servers, we can also remove the `OLS` prefix before the AppServerXXX constants, too. (we can put OLSAppServerXX to a later refactoring PR)

Reply:
Agreed. Removed the OLS prefix.